src/main/java/com/wolox/training/models/User.java

```diff
@@ -0,0 +1,100 @@
+package com.wolox.training.models;
+
+import com.wolox.training.exceptions.BookAlreadyOwnedException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "users")
+public class User {
+
+    public User() {
+    }
+
+    public User(String username, String name, LocalDate birthDate) {
+        this.username = username;
+        this.name = name;
+        this.birthDate = birthDate;
+    }
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE)
+    private long id;
+
+    @Column(nullable = false)
+    private String username;
+
+    @Column(nullable = false)
+    private String name;
+
+    @Column(nullable = false)
+    private LocalDate birthDate;
+
+    @ManyToMany(cascade = {CascadeType.PERSIST, CascadeType.MERGE})
```

Comment:
Here the relationships need to be REFRESH, PERSIST is ok

Persist says: If you persist the Users instance to the DB, then persist every item that this collection has in the DB as well.
Merge says: If you changed the in-memory values of the collection, then merge this changes to the DB so these are reflected there.
Refresh says: If something changed in the database values of the collection, then refresh those changes in the in-memory values.

Reply:
sorry my mistake the the relationships need to be REFRESH y MERGE